tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,246 @@
+.. _doc_xr_action_map:
+
+The XR action map
+=================
+
+Godot has an action map feature as part of the XR system. At this point in time this system is part of the OpenXR module but there are plans to encompass WebXR into this in the near future hence we call it the XR action map system in this document. It implements the build in action map system of OpenXR mostly exactly as it is offered.
+
+The XR action map system exposes input, positional data and output for XR controllers to your game. It does this by exposing named actions that can be tailored to your game and binding these to the actual inputs and outputs on your XR devices.
+
+As the XR action map is currently part of the OpenXR module, OpenXR needs to be enabled in your project settings to expose it:
+
+.. image:: img/openxr_settings.png
+
+
+
+You will then find the XR Action Map interface in the bottom of the screen:
+
+.. image:: img/xr_action_map.webp
+
+
+
+.. note::
+	Godots build in input system has many things in common with the XR action map system. In fact our original idea was to add functionality to the existing input system and expose the data to the OpenXR action map system. We may revisit that idea at some point but as it turns out there were just too many problems to overcome. To name a few:
+
+	 * Godots input system mainly centers around button inputs, XR adds triggers, axis, poses and haptics (output) into the mix. This would greatly complicate the input system with features that won't work for normal controllers or contrast with the current approach. It was felt this would lead to confusion for the majority of Godot users.
```

Comment:
```suggestion
    * Godot's input system mainly centers around button inputs, XR adds triggers, axis, poses and haptics (output) into the mix. This would greatly complicate the input system with features that won't work for normal controllers or contrast with the current approach. It was felt this would lead to confusion for the majority of Godot users.
```